content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-upgrade-phase.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Fase de atualização do kubeadm
+weight: 90
+content_type: conceito
+---
+
+Na versão v1.15.0, o kubeadm introduziu um suporte preliminar para as fases `kubeadm upgrade node`. Fases para outros subcomandos `kubeadm upgrade`, tal como `apply`, podem ser adicionadas nas seguintes versões.
+
+## Fase de atualização do nó do kubeadm {#cmd-node-phase}
```

Comment:
This header refers to an actual command (`kubeadm upgrade node phase`). Let's not translate that.